org-code-javabuilder/lib/src/main/java/org/code/javabuilder/CodeExecutionManager.java

```diff
@@ -170,4 +176,25 @@ private void onPostExecute() {
       this.isInitialized = false;
     }
   }
+
+  private void logMemoryUsage() {
```

Comment:
Nice! This seems like a helpful profiling tool - could we move this into a util in case we want to use it elsewhere?

Reply:
good idea, will update

Reply:
I moved it to a new util and removed the constant and method call here since it made less sense to call it now that it's in a separate util.